src/test/unit/services/subject.spec.js

```
@@ -0,0 +1,24 @@
+const subjectService = require('~/services/subject')
+const SubjectSpec = require('~/models/subject')
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)